torchmetrics/classification/dice.py

```diff
@@ -0,0 +1,156 @@
+# Copyright The PyTorch Lightning team.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from typing import Any, Dict, Optional
+
+from torch import Tensor
+
+from torchmetrics.functional.classification.dice import _dice_compute
+
+from torchmetrics.classification.stat_scores import StatScores  # isort:skip
```

Comment:
```suggestion
from torchmetrics.classification.stat_scores import StatScores
```
why is this needed?

Reply:
I copied this from [here](https://github.com/PyTorchLightning/metrics/blob/master/torchmetrics/classification/accuracy.py#L28).